packages/devtools_app/lib/src/network/network_request_inspector_views.dart

```diff
@@ -69,9 +69,9 @@ class HttpRequestHeadersView extends StatelessWidget {
             style: Theme.of(context).textTheme.subtitle2,
           ),
           Expanded(
-            child: Text(
+            child: SelectableText(
               value,
-              overflow: TextOverflow.ellipsis,
```

Comment:
can you add this back in? without this, the text will be cutoff for small screen widths, or if the splitter is dragged to the right

Reply:
Ellipsis won't work with it. 
SelectableText is an EditableText and overflows by scrolling horizontally or vertically if minLines or maxLines is set.
So it's basically a TextField that can't be edited.

@pedromassango please correct me if I'm wrong.

Reply:
Maybe the `NetworkRequestInspector` tab can have a minimum width to prevent text from wrapping extremely?

Reply:
Right. I though once https://github.com/flutter/flutter/pull/78927 is merged we would be able to set overflow for editable text widgets but there is still some work to be done to get this. I've filled https://github.com/flutter/flutter/issues/80434 to track those issues.

Reply:
Thanks!

Reply:
Filled https://github.com/flutter/flutter/issues/80435 to fix the overflow issue on `SelectableText` widget.